ui/pages/confirmations/hooks/useConfirmationNavigation.ts

```diff
@@ -128,8 +121,30 @@ export function navigateToConfirmation(
     return;
   }
 
-  if (CONNECT_APPROVAL_TYPES.includes(type)) {
```

Comment:
This was an extra render for no reason, since we know the approval type, we can navigate directly to the appropriate snap page.